docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -280,12 +280,12 @@ relation to port exhaustion.
 ## Queue (`queue` and `queue.*`)
 
 - `TYPE`: **persistable-channel**: General queue type, currently support: `persistable-channel`, `channel`, `level`, `redis`, `dummy`
-- `DATADIR`: **queues/**: Base DataDir for storing persistent and level queues. `DATADIR` for inidividual queues can be set in `queue.name` sections but will default to `DATADIR/`**`name`**.
+- `DATADIR`: **queues/**: Base DataDir for storing persistent and level queues. `DATADIR` for individual queues can be set in `queue.name` sections but will default to `DATADIR/`**`name`**.
 - `LENGTH`: **20**: Maximal queue size before channel queues block
 - `BATCH_LENGTH`: **20**: Batch data before passing to the handler
-- `CONN_STR`: **addrs=127.0.0.1:6379 db=0**: Connection string for the redis queue type.
-- `QUEUE_NAME`: **_queue**: The suffix for default redis queue name. Individual queues will default to **`name`**`QUEUE_NAME` but can be overriden in the specific `queue.name` section.
-- `SET_NAME`: **_unique**: The suffix that will added to the default redis
+- `CONN_STR`: **redis://127.0.0.1:6379/0**: Connection string for the redis queue type. Options can be set using query params. Similarly LevelDB options can also be set using: **leveldb://relative/path?option=value** or **leveldb:///absolute/path?option=value**
+- `QUEUE_NAME`: **_queue**: The suffix for default redis and disk queue name. Individual queues will default to **`name`**`QUEUE_NAME` but can be overriden in the specific `queue.name` section.
+- `SET_NAME`: **_unique**: The suffix that will addeds to the default redis
```

Comment:
```suggestion
- `SET_NAME`: **_unique**: The suffix that will be added to the default redis
```

It still does not make total sense to me. What does this alter? redis key names?

Reply:
OK, both levelDB and Redis are key-value dbs - meaning that the db is effectively a map `D[key] = value`

If you want to make a queue in such a DB, one relatively efficient way of doing this, is to use:

`D["HIGH"] = x_high` , `D["LOW"]= x_low` and `D[x] = value` - (with some compacting function that eventually moves everything back to 0 at some point)

now that would only allow you to have a one Queue per DB - so if you add a suffix you can have more you can add infinitely many more.

How do you represent an Unique Queue - well an Unique Queue is simply a list with a set. How do you represent a set? Simple:

`D[value] = 1` if the value is in the set.

If you add a suffix - which is sufficiently unique - then you can have multiple sets.

That's about the gist of it.

Reply:
I see you already updated it. It does certainly make some sense to me now even if I don't understand the mechanism completely, thanks. That's what documentation should be about.